src/server/routes/api/v1/parking-lot.ts

```diff
@@ -0,0 +1,113 @@
+import httpResponse from '../../../utils/http-response';
+import type {
+  FastifyError,
+  FastifyInstance,
+  FastifyRegisterOptions,
+  FastifyReply,
+  FastifyRequest,
+} from 'fastify';
+import { UnexistentParkingLotID } from '../../../error/parking-lot.service';
+
+export default function (
+  fastify: FastifyInstance,
+  _: FastifyRegisterOptions<unknown>,
+  done: (err?: FastifyError) => void,
+): void {
+  fastify.post(
+    '/',
+    async (
+      request: FastifyRequest<{
+        Body: {
+          name: string;
+        };
+      }>,
+      reply: FastifyReply,
```

Comment:
I'm still missing the schema here. Check how it is implemented for Users here:

https://github.com/whizzes/thruway-parking/blob/9050cba18896b6bb52ecdc3c84bf365247f58b74/src/server/routes/api/v1/users/index.ts#L21